vendor/assets/stylesheets/dvl/core/forms.scss

```diff
@@ -408,19 +409,23 @@ form {
   margin-bottom: $rhythm * 2;
 }
 
-.form_item_horiz_label {
+.form_item_horiz_label,
+.form_item_horiz legend {
```

Comment:
Any reason to not just apply the `form_item_horiz_label` class to the `<legend>` tag?